src/entities/Employee.js

```diff
@@ -0,0 +1,22 @@
+export default class Employee {
+  constructor () {
+    this._name = ""
+    this._position = ""
```

Comment:
De obicei, pentru functia unui angajat, se face o clasa separata (de ex; class Position), in care sa tii anumite detalii despre pozitie (nume, descriere, departament, etc). Ai putea face o clasa separata numita Position, care sa aiba o proprietate de name.